frontend/app/module/polling_stations/page/PollingStationListPage.tsx

```diff
@@ -0,0 +1,55 @@
+import { PollingStationType, usePollingStationList } from "@kiesraad/api";
+import { IconChevronRight } from "@kiesraad/icon";
+
+function pollingStationType(pollingStationType: PollingStationType): string {
+  switch (pollingStationType) {
+    case "Bijzonder":
+      return "Bijzonder";
+    case "Mobiel":
+      return "Mobiel";
+    case "VasteLocatie":
+      return "Vaste locatie";
+  }
+}
+
+export function PollingStationListPage() {
+  const { pollingStations } = usePollingStationList();
+
+  return (
+    <>
+      <header>
+        <section>
+          <h1>Stembureaus</h1>
+        </section>
+      </header>
+      <main>
+        <article>
+          <table id="polling_stations" className="polling_stations_table">
+            <thead>
+              <tr>
+                <th>Nummer</th>
```

Comment:
Should we consider a monospaced font for the numbers? Our `0` is significantly wider than the other integers. 
cc @jorisleker 

![image](https://github.com/user-attachments/assets/19c36164-24f2-4b3e-b21a-f7d0126ac747)

Reply:
This is actually the use case for [`font-variant-numeric: tabular-nums`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-numeric) but I don't know if our font supports that.

Reply:
Please discuss further in # https://github.com/kiesraad/abacus/issues/433 Create UI Table component